.eslintrc.js

```diff
@@ -79,7 +79,12 @@ module.exports = {
         'no-unused-vars': 0,
         'no-undef': 0,
         'no-use-before-define': 'off',
-        '@typescript-eslint/no-unused-vars': 'warn',
+        '@typescript-eslint/no-unused-vars': [
+          'warn',
+          {
+            argsIgnorePattern: '^_',
```

Comment:
I think we should ignore all unused arguments. I don't see a single place that wasn't an argument required by an interface that TypeScript's type checking enforced.

```suggestion
            args: 'none'
```
Sorry to take so long to get back to you on this @Korilakkuma. Would you mind undoing the changes and rebasing with this change?